src/modules/character.rs

```diff
@@ -113,6 +145,7 @@ mod test {
                 .config(toml::toml! {
                     [character]
                     success_symbol = "[➜](bold green)"
+                    root_symbol = "[➜](bold green)"
```

Comment:
can we have `root_success_symbol` and `root_error_symbol`?

Reply:
also for convention sake - can we change this to `#`

Reply:
Good call, how does it look now?